src/coreclr/nativeaot/System.Private.CoreLib/src/System/Threading/Thread.NativeAot.Windows.cs

```diff
@@ -26,6 +26,77 @@ public sealed partial class Thread
 
         partial void PlatformSpecificInitialize();
 
+        internal static void SleepInternal(int millisecondsTimeout)
+        {
+            Debug.Assert(millisecondsTimeout >= -1);
+
+            CheckForPendingInterrupt();
+
+            Thread currentThread = CurrentThread;
+            if (millisecondsTimeout == 0)
+            {
+                UninterruptibleSleep0();
```

Comment:
I do not see this `UninterruptibleSleep0` special case in regular CoreCLR, Should we make the logic the same here to avoid needless differences?